lib/rmg/CMapGenOptions.cpp

```diff
@@ -167,7 +172,10 @@ void CMapGenOptions::resetPlayersMap()
 		auto pc = PlayerColor(color);
 		player.setColor(pc);
 		auto playerType = EPlayerType::AI;
-		if ((getPlayerCount() != RANDOM_SIZE && color >= realPlayersCnt)
+		if (getPlayerCount() != RANDOM_SIZE && color < realPlayersCnt)
+		{
+			playerType = EPlayerType::HUMAN;
+		}else if ((getPlayerCount() != RANDOM_SIZE && color >= realPlayersCnt)
```

Comment:
please fix formatting
```suggestion
		}
		else if((getPlayerCount() != RANDOM_SIZE && color >= realPlayersCnt)
```